modules/rules_perl/metadata.json

```diff
@@ -11,10 +11,11 @@
         }
     ],
     "repository": [
-        "github:bazelbuild/rules_perl"
+        "github:bazel-contrib/rules_perl"
```

Comment:
Let's keep the old repo in the list so that validations for previous version keep working.